source/stylesheets/site.css.scss

```diff
@@ -74,11 +81,45 @@ a.brand-logo {
   color: $black;
 }
 
-.fc-unthemed td.fc-today {
-  background-color: $color;
+.fc-view, .fc-basic-view {
+  background-color: $lighttext;
+  color: $color;
+  border: 1px solid $black;
+  box-shadow: 2px 2px 2px $lighttext;
+}
+
+.fc-day-header {
+  background-color: $lighttext;
+  color: $black;
+  border: 1px solid $black !important;
```

Comment:
If you increased selector specificity you probably will not need the `!important`s